### PR TITLE
Update docker hacking information

### PR DESCRIPTION
**Pull request by luke-hill**
Take 3 :) This is a follow-on from #649 

NB: There are other links to travis in the HACKING.md file, but I don't know enough about this to fix it. So this is still needing to be updated. I've made an issue item about this to be tidied up

## Summary

Update a few review points leftover that were added by aslak after previous PR was merged

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).

## Checklist:

- [ ] The change has been ported to Java.
- [ ] The change has been ported to Ruby.
- [ ] The change has been ported to JavaScript.
- [ ] The change has been ported to Go.
- [ ] The change has been ported to .NET.
- [ ] I've added tests for my code.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: HACKING.md [hunk 1]
```diff
@@ -39,10 +39,11 @@ separated by an underscore. For example, `messages_gherkin-use-protobuf`.
 ### Local Testing / Development checks
 
 Cucumber runs with a highly coupled system of tests. Because cucumber is a mono-repo, testing these is best done
-in a docker container, due to needing a large amount of pre-requisites and often setting them up can be tricky.
+in a docker container, due to needing a large amount of pre-requisites. The setting up of these often can be a tricky
+task, so we use Docker to help us with this.
 
-To start with, we need to ensure we have Docker installed. This is usually fairly easy, and a comprehensive set
-of setup instructions are available [HERE](https://docs.docker.com/install/) for most OS'
+To start with, we need to ensure we have Docker installed. This is fairly straightforward, and a comprehensive set
```
Comment (mlvandijk) [inline HACKING.md]: Let's not use words like "easy", "simple", "straightforward" in any of the documentation.  (see also: https://github.com/cucumber/docs.cucumber.io/issues/389)

File: HACKING.md [hunk 2]
```diff
@@ -39,10 +39,11 @@ separated by an underscore. For example, `messages_gherkin-use-protobuf`.
...
+of setup instructions are available on the official docker site [HERE](https://docs.docker.com/install/) for most OS'
```
Comment (mlvandijk) [inline HACKING.md]: Maybe write "operating systems" instead of "OS'", because OS' looks weird...

**Comment by luke-hill** [comment]
Is this good to be merged? It's a slight improvement again...

@aslakhellesoy / @mlvandijk

**Comment by luke-hill** [comment]
I'm gonna merge this in. We can tidy things up later if needs be again. It's a slight improvement.

**Pull request merged**